server/channels/store/store.go

```diff
@@ -1115,7 +1115,7 @@ type ThreadMembershipOpts struct {
 // PostReminderMetadata contains some info needed to send
 // the reminder message to the user.
 type PostReminderMetadata struct {
-	ChannelId  string
+	ChannelID  string
```

Comment:
You also need to update the `SELECT` query at https://github.com/mattermost/mattermost/blob/73b14b417e748f1679be0acfa7f1cb4dc319ee5a/server/channels/store/sqlstore/post_store.go#L3242 to use `ChannelID`

Reply:
done!